app/poi_api/spec.yaml

```diff
@@ -31,6 +31,7 @@ components:
       type: "object"
       required:
         - suggestion_name
+        - submitted_by
```

Comment:
We actually don't need this attribute in the request body. We get the User submitting it form their JWT token. This is handled by our middleware function, which passes a User into your endpoint function so you know who's making the request